test/filters/all.test.js

```diff
@@ -79,11 +148,11 @@ describe('Filters', () => {
 
   describe('.renderSpec', () => {
     it('should work', async () => {
-      const doc = new AsyncAPIDocument({ asyncapi: '2.0.0', info: { title: 'dummy spec for testing', version: '1.5.73' } });
+      const doc = new AsyncAPIDocument({ asyncapi: '2.0.0', info: { title: 'dummy spec for testing', version: '1.5.34' } });
 
       const result = renderSpec(doc);
-      // check if '1.5.73' version is rendered
-      expect(result.includes('1.5.73')).toEqual(true);
+      // check if '1.5.34' version is rendered
+      expect(result.includes('1.5.34')).toEqual(true);
```

Comment:
Are you mocking me or 😂? I know it is test data, but when there is no need to change it, it just clutter the changes 😄